STL_Extension/include/CGAL/assertions.h

```diff
@@ -63,9 +63,9 @@ namespace CGAL {
 // =====================
 // failure functions
 // -----------------
-CGAL_EXPORT void assertion_fail      ( const char*, const char*, int, const char* = "") CGAL_NORETURN;
-CGAL_EXPORT void precondition_fail   ( const char*, const char*, int, const char* = "") CGAL_NORETURN;
-CGAL_EXPORT void postcondition_fail  ( const char*, const char*, int, const char* = "") CGAL_NORETURN;
+CGAL_EXPORT CGAL_NORETURN void assertion_fail      ( const char*, const char*, int, const char* = "") ;
+CGAL_EXPORT CGAL_NORETURN void precondition_fail   ( const char*, const char*, int, const char* = "") ;
+CGAL_EXPORT CGAL_NORETURN void postcondition_fail  ( const char*, const char*, int, const char* = "") ;
```

Comment:
Have you tested? Seeing the patch, I have a doubt: en Windows, `CGAL_EXPORT` is also a `__declspec` (if you compile CGAL as a DLL), and I think there could be only one of them, combining the two attributes.

Reply:
I hadn't checked together with CGAL_EXPORT, but did it now, and it works.